templates/rhpam73-authoring-ha.yaml

```diff
@@ -1189,8 +1189,10 @@ objects:
             value: "${APPLICATION_NAME}-mysql"
           - name: RHPAM_SERVICE_PORT
             value: "3306"
+          - name: TIMER_SERVICE_DATA_STORE
```

Comment:
You don't need to set it, it is automatically configured by the EJB timer scripts.
https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh#L88-L100

The reason we set it on the script is to make sure that the datasource used to configure the datastore will be the ejb timer datasource.

Reply:
Ok, so the TIMER_SERVICE_DATA_STORE can be removed from all our templates then?

Reply:
yes, it will be overridden anyways.